source/docs/03-metadata.xml

```diff
@@ -1382,156 +1354,308 @@
                   </div>
-                  <div xml:id="foliumdimensions" type="div4">
-                     <head>Specifying page dimensions</head>
-                     <p>Within <gi scheme="MEI">surface</gi>, each <gi scheme="MEI">graphic</gi> element may specify its dimensions using the following attributes:</p>
+                  <div xml:id="physicalProperties" type="div4">
+                     <head>Physical Properties</head>
+                     <p>Even if the section <ptr target="#msdesc"/> is about how do a physical description this part is explaining elements on the medium level.</p>
                      <p>
                         <specList>
-                           <specDesc key="att.height" atts="height"/>
-                           <specDesc key="att.width" atts="width"/>
+                           <specDesc key="extent"/>
+                           <specDesc key="dimensions"/>
+                           <specDesc key="dim"/>
+                           <specDesc key="watermark"/>
+                           <specDesc key="stamp"/>
+                           <specDesc key="physMedium"/>
+                           <specDesc key="scoreFormat"/>
                         </specList>
                      </p>
-                     <p>The values of those attributes, however, specify the height and width of the digital resource, the scan of the source, and they are typically given in pixels (see <ptr target="#facsimileElements"/>). In contrast, <gi scheme="MEI">folium</gi> and <gi scheme="MEI">bifolium</gi> are supposed to provide the dimensions of the original source in cm or inches. This makes it possible to combine separate parts of a manuscripts stored in different libraries, which are scanned at different resolutions. In case of <gi scheme="MEI">bifolium</gi> elements, these dimensions apply to the <hi rend="italic">folded</hi> sheet.</p>
-                     <p>Some printed scholarly editions like the <hi rend="italic">Neue Bach-Ausgabe</hi> provide very detailed information about the sizes and binding of individual leaves of a manuscript; with <gi scheme="MEI">foliaDesc</gi> and its children it is possible to encode that information in processable ways, even without providing digitizations of the sources via <gi scheme="MEI">surface</gi>.</p>
+                     <p>Extention and Dimension</p>
+                     <p>Watermark</p>
+                     <p>Stamp</p>
+                     <p>physMedium</p>
```

Comment:
Should be wrapped in `gi`?

Reply:
Those are placeholders that will be substituted with content immediately after this PR is merged. It will be done before the PR to develop will be opened. However, should I use `gi` here? If yes, just tell me and I'll do :)

Reply:
@riedde I would suggest to keep these changes for the next PR.

Reply:
I'm not convinced by this -- I'm not sure why an element name is listed in the same list as non-element names? At the very least can we change this to "Medium"? Or something?